src/tests/app.test.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-unused-expressions */
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import store from '../redux/configureStore';
+import App from '../App';
+
+test('It should Should display the header', () => {
+  render(
+    <Provider store={store}>
+      <Router>
+        <App />
+      </Router>
+    </Provider>,
+  );
+  const headerText = screen.getByText(/COVID-19 CASE STATISTICS/i);
+  expect(headerText).toBeInTheDocument;
+});
```

Comment:
There is just one test case 

>![Screenshot (337)](https://user-images.githubusercontent.com/66112760/147077102-c09fb026-65d3-4596-a148-324f8e2fdadd.png)

Let's provide more integration tests using React Testing Library ✊

Reply:
You needed to press a to run all the tests